README.md

```diff
@@ -1,2 +1,53 @@
 # MoSDeF-dihedral-fit
-MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software
+MoSDeF-dihedral-fit: A simple software package to fit dihedrals via the MoSDeF software.
+
+This package is a tool to add dihedral parameters for [Ryckaert-Bellemans](https://manual.gromacs.org/current/reference-manual/functions/bonded-interactions.html#proper-dihedrals-ryckaert-bellemans-function) dihedral based forcefields by fitting QM calculations for the rotation about the dihedral angle to this functional form. The main functionality is found through the following code snippet.
```

Comment:
This should not say RB only.  It should say the same as in the docs or similar.  

"The **MoSDeF-dihedral-fit** program is an open-source, transparent, and lightweight Python software package capable
of fitting dihedrals for existing forces fields.  This software fits the
`Optimized Potentials for Liquid Simulations (OPLS) <https://pubs.acs.org/doi/10.1021/ja9621760>`_ style
dihedrals, then also analytically converts them to the periodic/`CHARMM <https://www.charmm.org>`_ dihedral and
Ryckaert-Bellemans (RB) torsion.."

Reply:
Yeah this is fair. Maybe we should show an example somewhere on the README of the file outputs that you might get from a fit.